package.json

```diff
@@ -21,25 +21,25 @@
     "clean": "git clean -xdf"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^14.0.3",
-    "@angular-eslint/builder": "^14.0.0",
-    "@angular-eslint/eslint-plugin": "14.0.0",
-    "@angular-eslint/eslint-plugin-template": "14.0.0",
-    "@angular-eslint/schematics": "14.0.0",
-    "@angular-eslint/template-parser": "14.0.0",
-    "@angular/animations": "^14.0.3",
-    "@angular/cli": "^14.0.3",
-    "@angular/common": "^14.0.3",
-    "@angular/compiler": "^14.0.3",
-    "@angular/compiler-cli": "^14.0.3",
-    "@angular/core": "^14.0.3",
-    "@angular/forms": "^14.0.3",
-    "@angular/platform-browser": "^14.0.3",
-    "@angular/platform-browser-dynamic": "^14.0.3",
-    "@angular/router": "^14.0.3",
+    "@angular-devkit/build-angular": "^15.2.8",
```

Comment:
(if) it's compatible with 14 then I'm not seeing any issues with upgrading to 15, however, if it is compatible why do these "need" to be updated?